cinema/views.py

```diff
@@ -127,20 +133,46 @@
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+    @extend_schema(
```

Comment:
The `extend_schema` decorator is correctly used to define the schema for the `list` action in `MovieViewSet`, providing detailed API documentation.